pages/clubs/index.jsx

```diff
@@ -34,6 +37,9 @@ var ClubsGuides = React.createClass({
         <HeroUnit>
           <h1>Mozilla Clubs</h1>
           <h2>Local groups that read, write, and participate on the web</h2>
+          <div className="single-button hero-button">
+            <a className="btn" onClick={this.showApplication} href="#">{this.context.intl.formatMessage({id: 'apply_to_be_captain_link'})}</a>
```

Comment:
this really should be a `<button>`, not an `<a>` - it's not a page navigation element